dist/svg.js

```diff
@@ -553,16 +553,28 @@ SVG.extend(SVG.PointArray, {
     // if already is an array, no need to parse it
     if (Array.isArray(array)) return array
 
-    // split points
-    array = this.split(array)
 
     // parse points
-    for (var i = 0, il = array.length, p, points = []; i < il; i++) {
-      p = array[i].split(',')
-      points.push([parseFloat(p[0]), parseFloat(p[1])])
-    }
+    var points = array.split(/\s|,/)
+
+    // validate points - https://svgwg.org/svg2-draft/shapes.html#DataTypePoints
+    // Odd number of coordinates is an error. In such cases, drop the last odd coordinate.
+    if (points.length % 2 !== 0) points.pop()
```

Comment:
Yes sounds right so far. My intention is, that users find mistakes in their js code not in their svg code. Anyway I guess its fine